pages/manage_and_operate/iam/iam-logs-forwarding/guide.en-gb.md

```diff
@@ -0,0 +1,222 @@
+---
+title: OVHcloud Account logs Forwarding to LDP
+excerpt: "Forwarding OVHcloud Account logs to Logs Data Platform"
+updated: 2023-10-23
+---
+
+ 
+
+## Objective
+
+In this guide, you will learn how to transform your OVHcloud Account events as actionable data through Logs Data Platform, a fully managed solution that helps you store, archieve, query and visualize your logs.  
+To discover Logs Data Platform before continuing this guide, please refer to [this documentation](/pages/manage_and_operate/observability/logs_data_platform/getting_started_introduction_to_LDP).
+
+## Requirements
+
+- An OVHcloud Account
+- a Logs Data Platform (LDP) account within this OVHcloud Account with at least one active stream configured. [This guide](/pages/manage_and_operate/observability/logs_data_platform/getting_started_quick_start) will walk you through all the necessary steps
+
+## Glossary
+
+**Logs Data Platform**: fully managed and secured log management platform proposed by OVHcloud. Find more information on the [Logs Data Platform service page](https://www.ovhcloud.com/en-ie/logs-data-platform/).
+
+**Data Stream**: a logical partition of logs that you create in an LDP account that you will use when ingesting, visualizing or querying your logs. Multiple sources can be stored in the same data stream, and it is the unit for defining a logs pipeline (retention policy, archiving, live streaming...), access rights and alert policies.
+
+**Logs forwarding**: feature integrated in an OVHcloud product to ingest logs from its services to a data stream of an LDP account in the same OVHcloud account. The feature has to be enabled by the customer per service.
+
+**Logs forwarding Subscription**: when enabling the logs forwarding for a given OVHcloud service to a given LDP data stream, a subscription is created and attached to the data stream for further management by the customer.
+
+**OVH Account audit logs**: Log entries for sensible activities happening within your OVH Account customer main need to collect and use for compliance purposes and other business policy enforcement.
+
+**OVH Account activity logs**: Log entries for all API calls done through OVHCloud API including those triggered when using OVHcloud manager.
+
+**OVH Account policy logs**: Log entries for OVH services access acceptance or denied access due to of a security policy evaluation.
+
+## Instruction
+
+### OVHcloud Account logs type
+
+OVHcloud account propose 3 level of logs
+
+- **Audit Logs**: provide a security-relevant, chronological set of records documenting the sequence of actions in your OVHcloud Account. (ie. logins, password change, …)
```

Comment:
duplicate from glossary. Maybe you should just keep logs level definition only here.

Reply:
Can we have the exhaustive list of actions that will be describe in theses logs?